api/proto/teleport/integration/v1/awsoidc_service.proto

```diff
@@ -32,6 +36,48 @@ service AWSOIDCService {
   rpc DeployDatabaseService(DeployDatabaseServiceRequest) returns (DeployDatabaseServiceResponse);
 }
 
+// ListEICERequest is a request for a paginated list of AWS EC2 Instance Connect Endpoints.
+message ListEICERequest {
+  // Integration is the AWS OIDC Integration name.
+  string integration = 1;
+  // Region is the AWS Region
+  string region = 2;
+  // VPCIDs is used to filter for EICEs of those VPCs.
+  repeated string vpc_ids = 3;
+  // NextToken is the token to be used to fetch the next page.
+  // If empty, the first page is fetched.
+  string next_token = 4;
```

Comment:
Are these all mandatory?

Reply:
All except for `next_token`.
I added a comment to `next_token`, expecting that the reader would assume that if I'm being explicit about `next_token` being optional, one would assume the others were required.

I guess it was not effective :sweat_smile: 
Should I add a comment to `integration,region,vpc_ids` fields?